src/plugins/data_view_management/public/components/edit_index_pattern/index_header/index_header.tsx

```diff
@@ -44,19 +46,44 @@ const removeTooltip = i18n.translate('indexPatternManagement.editDataView.remove
   defaultMessage: 'Delete',
 });
 
+const refreshAriaLabel = i18n.translate('indexPatternManagement.editDataView.refreshAria', {
+  defaultMessage: 'Refresh',
+});
+
+const refreshTooltip = i18n.translate('indexPatternManagement.editDataView.refreshTooltip', {
+  defaultMessage: 'Refresh local copy of data view field list',
+});
+
+const refreshLabel = i18n.translate('indexPatternManagement.editDataView.refreshLabel', {
+  defaultMessage: 'Refresh',
```

Comment:
Since we already had "Refresh" button in the earlier Kibana versions and it had a different purpose, using a label other than "Refresh" might work better here. For example, "Clear fields cache".

Reply:
I think my concern with `Clear fields cache` is that its unclear where the cache lives. The tooltip explains in detail.

Reply:
@amyjtechwriter I'm curious about your thoughts on this.

Reply:
It would be great to make it clear that it's about Kibana and browser cache rather than something to do with how elasticsearch itself manages fields.

Reply:
@amyjtechwriter My concern with those phrases, and you're free to overrule me - in the context of a web browser they mean _everything_ is cleared. The button reloads only this specific data view field set. 

Past that, I don't think we should worry about button length.